pkgs/development/tools/convco/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, openssl, perl, pkg-config, rustPlatform
```

Since `stdenv` is unused, we can get rid of it!

Additionally, we should move `rustPlatform` higher in the order as a result of the recently-merged https://github.com/NixOS/nixpkgs/pull/110060 (which boils down to: "arguments should be listed in the order they are used, with the exception of `lib` which should always go first").

```suggestion
{ lib, rustPlatform, fetchFromGitHub, openssl, perl, pkg-config
```

---

Stdenv needs to be used for `isDarwin` sadly!

---

Maybe it's just me, but I don't see any `isDarwin` usage ;)